src/object/obj_rpc.c

```diff
@@ -65,39 +55,9 @@ crt_proc_daos_key_desc_t(crt_proc_t proc, daos_key_desc_t *key)
 }
 
 static int
-crt_proc_daos_obj_id_t(crt_proc_t proc, daos_obj_id_t *doi)
+crt_proc_daos_unit_oid_t(crt_proc_t proc, daos_unit_oid_t *p)
```

Comment:
for things that are declared as CRT_RAW why do we still need to have proc function if all proc functions have same view of   'return crt_proc_memcpy(proc, p, sizeof(*p))' ? why not generate it automatically?

Reply:
I have to keep this because of we have "CRT_ARRAY" instance which is use this routine.

Reply:
Automatic generation is problematic because of struct can have pointers which should be handled properly.

Reply:
I was thinking that CRT_RAW implies it can be copied directly for which we could generate copy function automatically and CRT_VAR are then the ones that need to have custom proc function to handle pointers and other fields that cannot be simply memcpy-ed